src/main/webapp/WEB-INF/tags/instructor/results/filterPanel.tag

```diff
@@ -1,103 +1,95 @@
-<%@ tag description="instructorFeedbackResultsTop - Filter Panel" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/results" prefix="r" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel" %>
```

Comment:
Same here - move the description back to the top. It is the results (prefix r) that should be moved below functions (prefix fn).